app/bg/dbs/settings.js

```diff
@@ -44,7 +45,12 @@ export const setup = async function (opts) {
     default_dat_ignore: '.git\n.dat\nnode_modules\n*.log\n**/.DS_Store\nThumbs.db\n',
     analytics_enabled: 1,
     dat_bandwidth_limit_up: 0,
-    dat_bandwidth_limit_down: 0
+    dat_bandwidth_limit_down: 0,
+    selected_search_engine: 0,
+    default_search_engines: [
```

Comment:
We should probably call this `search_engines` instead of `default_search_engines` since it's user-modifiable